.gitignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+dist/
+.rpt2_cache/
+yarn-error.log
+coverage
```

Comment:
[Minor] Add `/` to identify that this is a directory - it's not needed but just for better readability.